Kernel/System/DynamicField/ObjectType/Ticket.pm

```diff
@@ -289,7 +289,7 @@ sub ObjectDataGet {
         Param => 'TicketID',
     );
 
-    return if !$TicketID;
+    return unless $TicketID;
```

Comment:
Cool, ich bin ein Fan von 'return unless'

Reply:
You got me hooked on that 😁